frontend/src/v5/ui/components/dashboard/dashboardList/useOrderedList/useOrderedList.hooks.ts

```diff
@@ -38,6 +38,10 @@ export const useOrderedList = <T>(items: T[], defaultConfig: ISortConfig) => {
 				return a[column].getTime() - b[column].getTime();
 			}
 
+			if (a[column] instanceof File) {
+				return a[column].name.localeCompare(b[column].name);
+			}
```

Comment:
this seems a bit custom, can it be generalised? or change the useOrderList.hooks to something that cater to this need?